tooling/templatize/pkg/pipeline/shell.go

```diff
@@ -287,9 +287,11 @@ func checkCachedOutput[T any](logger logr.Logger, data any, stepCacheDir string)
 
 		var output T
 		if err := json.Unmarshal(content, &output); err != nil {
-			return "", nil, nil, fmt.Errorf("failed to deserialize output: %w", err)
+			logger.V(2).Info("Failed to deserialize cached output, ignoring cache", "err", err, "content", string(content))
+			// Don't fail, just proceed without cache
```

Comment:
Please no. Figure out what corrupted it and fix it.

Reply:
Agreed, this was identified as a bug.